invenio_users_resources/services/users/config.py

```diff
@@ -46,7 +46,13 @@ class UserSearchOptions(SearchOptions, SearchOptionsMixin):
     }
     query_parser_cls = QueryParser.factory(
         tree_transformer_cls=SearchFieldTransformer,
-        fields=["username^2", "email^2", "profile.full_name^3", "profile.affiliations"],
+        fields=[
+            "username^2",
+            "email^2",
+            "id^3",
```

Comment:
@lnielsen @ntarocco @zzacharo if I remember correctly in the past we decided not to expose the search by user ID, did something change ?

Reply:
We **Must** protect the search by id because of security reasons.

Reply:
There's two parts to this.......as a normal user no you should not be able to search on id......as an admin yes, you should be able to search on id.......easiest my be to use two different endpoints

Reply:
did you check if setting field_dump_permissions on user schema is enough ?